synapse/storage/databases/main/relations.py

```diff
@@ -676,7 +691,7 @@ async def get_bundled_aggregations(
         results = {}
         for event in events:
             event_result = await self._get_bundled_aggregation_for_event(event, user_id)
-            if event_result is not None:
+            if event_result:
```

Comment:
This change means that if there are no bundled aggregations we don't include it at all, instead of including an empty object.

This was always the intention and what all the docstrings say, but guess the logic was slightly wrong. I noticed this when I was debugging something else.